help

> [21:20:04] [Server thread/INFO]: ajan_12 issued server command: /sffix
> [21:20:04] [Server thread/INFO]: Tried to fix items but none found.
> [21:20:04] [Server thread/ERROR]: null
> org.bukkit.command.CommandException: Unhandled exception executing command 'sffix' in plugin SlimefunFix v1
> 	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:46) ~[spigot.jar:git-Spigot-2086bb0-1988d7f]
> 	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:141) ~[spigot.jar:git-Spigot-2086bb0-1988d7f]
> 	at org.bukkit.craftbukkit.v1_12_R1.CraftServer.dispatchCommand(CraftServer.java:648) ~[spigot.jar:git-Spigot-2086bb0-1988d7f]
> 	at net.minecraft.server.v1_12_R1.PlayerConnection.handleCommand(PlayerConnection.java:1397) [spigot.jar:git-Spigot-2086bb0-1988d7f]
> 	at net.minecraft.server.v1_12_R1.PlayerConnection.a(PlayerConnection.java:1232) [spigot.jar:git-Spigot-2086bb0-1988d7f]
> 	at net.minecraft.server.v1_12_R1.PacketPlayInChat.a(PacketPlayInChat.java:45) [spigot.jar:git-Spigot-2086bb0-1988d7f]
> 	at net.minecraft.server.v1_12_R1.PacketPlayInChat.a(PacketPlayInChat.java:1) [spigot.jar:git-Spigot-2086bb0-1988d7f]
> 	at net.minecraft.server.v1_12_R1.PlayerConnectionUtils$1.run(SourceFile:13) [spigot.jar:git-Spigot-2086bb0-1988d7f]
> 	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:?]
> 	at java.util.concurrent.FutureTask.run(Unknown Source) [?:?]
> 	at net.minecraft.server.v1_12_R1.SystemUtils.a(SourceFile:46) [spigot.jar:git-Spigot-2086bb0-1988d7f]
> 	at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:748) [spigot.jar:git-Spigot-2086bb0-1988d7f]
> 	at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:406) [spigot.jar:git-Spigot-2086bb0-1988d7f]
> 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:679) [spigot.jar:git-Spigot-2086bb0-1988d7f]
> 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:577) [spigot.jar:git-Spigot-2086bb0-1988d7f]
> 	at java.lang.Thread.run(Unknown Source) [?:?]
> Caused by: java.lang.ArrayIndexOutOfBoundsException: 0
> 	at top.ageofelysian.SlimefunFix.SlimefunFix.onCommand(SlimefunFix.java:119) ~[?:?]
> 	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:44) ~[spigot.jar:git-Spigot-2086bb0-1988d7f]
> 	... 15 more

--- a/SlimefunFix/src/main/java/top/ageofelysian/SlimefunFix/SlimefunFix.java
+++ b/SlimefunFix/src/main/java/top/ageofelysian/SlimefunFix/SlimefunFix.java
@@ -105,6 +105,7 @@ public final class SlimefunFix extends JavaPlugin {
                             } else {
                                 Bukkit.getLogger().info("Tried to fix items but none found.");
                             }
+                            return true;
                         }
                     } else {
                         player.sendMessage(ChatColor.DARK_RED + "You do not have enough permissions to perform this command.");

